terria/africa.json

```diff
@@ -235,86 +234,97 @@
-                     "opacity": 1,
-                     "featureTimesProperty": "data_available_for_dates"
+                     "name": "All time water summary source data",
+                     "type": "group",
+                     "preserveOrder": true,
+                     "items": [
+                        {
+                           "name": "Count of wet observations",
+                           "type": "wms",
+                           "layers": "wofs_2_summary_wet",
+                           "url": "https://ows.digitalearth.africa/",
+                           "linkedWcsUrl": "https://ows.digitalearth.africa/",
+                           "linkedWcsCoverage": "wofs_2_summary_wet",
+                           "ignoreUnknownTileErrors": true,
+                           "opacity": 1
+                        },
+                        {
+                           "name": "Count of clear observations",
+                           "type": "wms",
+                           "layers": "wofs_2_summary_clear",
+                           "url": "https://ows.digitalearth.africa/",
+                           "linkedWcsUrl": "https://ows.digitalearth.africa/",
+                           "linkedWcsCoverage": "wofs_2_summary_clear",
+                           "ignoreUnknownTileErrors": true,
+                           "opacity": 1
+                        }
+                     ]
                   }
                ]
-            },
+            }
+         ]
+      },
+      {
+         "name": "Vegetation",
+         "type": "group",
+         "preserveOrder": true,
+         "items": [
             {
-               "name": "Cloud Free Mosaic (Provisional)",
-               "type": "group",
-               "preserveOrder": true,
-               "items": [
-                  {
-                     "name": "Geometric median",
-                     "type": "wms",
-                     "layers": "ga_ls8c_gm_2_annual",
-                     "url": "https://ows.digitalearth.africa/",
-                     "linkedWcsUrl": "https://ows.digitalearth.africa/",
-                     "linkedWcsCoverage": "ga_ls8c_gm_2_annual",
-                     "ignoreUnknownTileErrors": true,
-                     "opacity": 1,
-                     "featureTimesProperty": "data_available_for_dates"
-                  }
-               ]
+               "name": "Individual fractional cover (Development)",
+               "type": "wms",
+               "layers": "ls_usgs_fc_scene",
+               "url": "https://ows.digitalearth.africa/",
+               "linkedWcsUrl": "https://ows.digitalearth.africa/",
+               "linkedWcsCoverage": "ls_usgs_fc_scene",
+               "ignoreUnknownTileErrors": true,
+               "opacity": 1,
+               "featureTimesProperty": "data_available_for_dates"
             }
          ]
       },
       {
-         "name": "Ancillary Data",
+         "name": "Elevation",
```

Comment:
The reason for calling this ancillary is that we may include other ancillary data here.

For example, the external surface water dataset may go here?

Reply:
From Slack:"
Thanks Alex appreciate the input, these changes were done with input from Fang, Negin and Chad, and Adam reviewed. We've also done work on rewriting the meta data for each, so for where we've simplified naming we have explained these in more detail within the data descriptions"

Reply:
The general idea with the naming of these top-level headings is to group datasets by themes. So you'd have a `Water` heading where anything relating to water would go, a `Satellite Images` section where all surface reflectance goes, etc.  Using this convention, it makes sense to have an `Elevation` heading where all datasets with height data can go.